Gordon360/Services/NewsService.cs

```diff
@@ -6,16 +6,26 @@
 using System;
 using System.Collections.Generic;
 using System.Data.SqlClient;
+using System.IO;
+using System.Linq;
+using System.Web;
 
 namespace Gordon360.Services
 {
     public class NewsService : INewsService
     {
         private IUnitOfWork _unitOfWork;
 
+        private CCTEntities1 _context;
+
+        private static readonly string UploadsFolder = "/browseable/uploads/news/";
+        private readonly string FolderPath = HttpContext.Current.Server.MapPath("~" + UploadsFolder);
```

Comment:
It's odd to have both of these variables, when `UploadsFolder` is never used except in deriving `FolderPath`. Perhaps a clearer, simpler alternative would be:
```suggestion
        private readonly string NewsUploadsPath = HttpContext.Current.Server.MapPath("~/browseable/uploads/news/");
```